samples/templates/express-direct-auth/web-server/routes/logout.js

```diff
@@ -14,6 +14,7 @@ router.post('/logout', async (req, res) => {
     await authClient.revokeAccessToken();
     // Clear local session
     req.session.destroy();
+    // TODO: only clear okta session for custom AS
```

Comment:
could you explain this a little bit?

Reply:
Make a bookmark here first, looks like right now the logout redirect keep failing for the org AS. I remember someone mentioned it's not supported for the org AS, let me check with the team.